### PR TITLE
Student Scores: adds tooltips, ellipsis and scale/wrap to tabs

### PR DESCRIPTION
**Pull request by darrenmason**
tab-truncate

regarding task: https://www.pivotaltracker.com/story/show/112365737

added tooltips, ellipsis and scale/wrap to tabs to allow for longer names and more periods

### CODE REVIEW

**Comment by philschatz** [comment]
Is the tab width (and ellipses) not possible using just CSS?

**Comment by philschatz** [comment]
For example, something like:

```css
.period-tab {
  display: inline-block;
  max-width: 100px;
  overflow-x: hidden;
  text-overflow: ellipses;

  white-space: nowrap;
}
```


Otherwise, can you add tests?

File: resources/styles/mixins.less [hunk 1]
```diff
@@ -80,6 +82,14 @@
     a {
       font-weight: 400;
       color: @tutor-neutral !important;
+      span {
```
Comment (philschatz) [inline resources/styles/mixins.less]: Could you add a class to this element so it is not selecting `span`? Maybe `react-bootstrap` does not have an easy way to do this though

File: resources/styles/mixins.less [hunk 2]
```diff
@@ -71,7 +71,9 @@
 
 .tutor-nav-tabs() {
   background: none;
-  li {
+  display: flex;
+  flex-wrap: wrap;
+  li.tab-item-period {
     text-align: center;
```
Comment (nathanstitt) [inline resources/styles/mixins.less]: tutor-nav-tabs are used on other things than periods. ;)

**Pull request merged**